libcld2.yaml

```diff
@@ -0,0 +1,107 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: libcld2
+  description: Compact Language Detector 2
+  url: https://github.com/CLD2Owners/cld2
+  version: 0_git20240911
+  epoch: 0
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - glibc-dev
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - libtool
+      - wolfi-baselayout
+  environment:
+    CXXFLAGS: "-fPIC"
+    LDFLAGS: "-Wl,-soname,libcld2.so.0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b56fa78a2fe44ac2851bae5bf4f4693a0644da7b
+      repository: https://github.com/CLD2Owners/cld2
+      branch: master
+
+  # https://salsa.debian.org/debian/cld2/-/tree/master/debian/patches
+  - uses: patch
+    with:
+      series: series
+
+  - working-directory: internal
+    pipeline:
+      - runs: |
+          ./compile_libs.sh
+          install -Dm755 libcld2.so "${{targets.destdir}}"/usr/local/lib/libcld2.so
+          install -Dm755 libcld2_full.so "${{targets.destdir}}"/usr/local/lib/libcld2_full.so
+          mkdir -p "${{targets.destdir}}"/usr/local/include/cld2/internal
+          install -Dm644 *.h "${{targets.destdir}}"/usr/local/include/cld2/internal
+
+  - working-directory: public
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.destdir}}"/usr/local/include/cld2/public
+          install -Dm644 *.h "${{targets.destdir}}"/usr/local/include/cld2/public
+
+  - uses: strip
+
+subpackages:
+  - name: "libcld2-static"
+    description: "libcld2 static libraries"
+    pipeline:
+      - uses: split/static
+
+  - name: "libcld2-dev"
+    description: "libcld2 development headers"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libcld2
+
+update:
+  enabled: false
+  exclude-reason: No releases or tags
```

Comment:
this is perfect case where you can use git: {} and schedule for example: https://github.com/wolfi-dev/os/pull/26391

Reply:
TIL! That's great.